docker-compose.yml

```diff
@@ -40,6 +40,8 @@ services:
         - 8081:8081
       env_file:
         - .env
+      environment:
+        - API_ROOT=http://localhost:8080
```

Comment:
This should go in .env, no?

Reply:
I want to expose it more clearly as it will only change for the deployed version

Reply:
Wait, no I disagree with this. This breaks non-Docker `yarn watch`.

We should either put all env vars in .env, or in `environment` (this depends on https://github.com/bayesimpact/tds/issues/159). In the meantime, `yarn watch` will be broken.

Reply:
Hmm good catch, fixed in #161